api/src/main/java/org/openmrs/api/db/ConditionDAO.java

```diff
@@ -53,6 +53,15 @@
 	 */
 	List<Condition> getActiveConditions(Patient patient);
 
+	/**
+	 * Gets all conditions related to the specified patient.
+	 *
+	 * @param patient the patient whose conditions are being queried.
```

Comment:
Instead of having a duplicate of the javadoc here, you would use the @see annotation to point to the one at the service interface.

Reply:
@dkayiwa  I updated the PR. Please take a look :)